images/base/Dockerfile

```diff
@@ -12,6 +12,8 @@ FROM centos:centos7
 # base images land
 RUN yum swap -y -- remove systemd-container\* -- install systemd systemd-libs
 
-RUN yum install -y git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
+RUN yum install --setopt=tsflags=nodocs -y git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
     socat ethtool device-mapper iptables && \
-    yum clean all
+    yum clean all && \
+    rpm -V git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
```

Comment:
I was thinking env stored in the image may be handy for other uses like verifying the image after it's built but I don't think that's strong enough argument to add a useless layer.